src/data/tests/test_data_file_handler.py

```diff
@@ -0,0 +1,68 @@
+# Python Imports
+import os
+import unittest
+import numpy as np
+import pandas as pd
+from pathlib import Path
+from result import Ok, Err
+from unittest.mock import patch
+
+# Project Imports
+from src.data.data_file_handler import DataFileHandler
+
+
+class TestDataFileHandler(unittest.TestCase):
+
+    def setUp(self):
+        self.current_directory = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Microimprovement (maybe): If this `self.current_directory` is used as a constant, that is, its value won't change throughout the tests, you can instead initialise it in
```py
@classmethod
def setUpClass(cls):
    cls.current_directory = ...
```
This method is only called once, on class initialisation, contrary to `setUp` which is called once for every case.

Reply:
For some reason, I thought what I had was doing that per class, not per test. Ty!